src/BulletCollision/Gimpact/btBoxCollision.h

```diff
@@ -234,7 +234,24 @@ ATTRIBUTE_ALIGNED16	(class) btAABB
 
 	btAABB()
 	{}
-
+	
```

Comment:
Those operators don't make much sense, why do we need them and who is using them?